core/arch/arm/plat-imx/conf.mk

```diff
@@ -306,11 +307,23 @@ CFG_PM_STUBS ?= y
 supported-ta-targets = ta_arm64
 endif
 
+CFG_TZC380 ?= y
+
+ifneq (,$(filter y, $(CFG_IMX8MM) $(CFG_IMX8MQ)))
+CFG_TZC_NSEC_START = 0x0
+CFG_TZC_SEC_START = CFG_TZDRAM_START - CFG_DRAM_BASE
+CFG_TZC_SHMEM_START = CFG_SHMEM_START - CFG_DRAM_BASE
```

Comment:
`?=`

Reply:
here `CFG_TZC_*_START` look like offsets. It is what `tzc_auto_configure()` expect?

Reply:
@jforissier I will fix it
@etienne-lms Yep. Region start addresses for the TZASC on imx8 are the start address minus the DRAM base address